paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -636,6 +647,7 @@ set_tests_properties(test_analyzer_googlenet PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_small_dam PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_transformer PROPERTIES TIMEOUT 120)
 set_tests_properties(test_analyzer_bert PROPERTIES TIMEOUT 120)
+set_tests_properties(test_analyzer_multi_model_prediction PROPERTIES TIMEOUT 120)
```

Comment:
```
2020-12-08 23:59:52 0. You must have one QA (XieYunshen(Recommend) or chalsliu) approval for setting parameter RUN_TYPE as EXCLUSIVE, DIST, NIGHTLY, EXCLUSIVE:NIGHTLY or DISTNIGHTLY, or setting TIMEOUT properties.
2020-12-08 23:59:52 The corresponding lines are as follows:
2020-12-08 23:59:52 paddle/fluid/inference/tests/api/CMakeLists.txt
2020-12-08 23:59:52 +set_tests_properties(test_analyzer_multi_model_prediction PROPERTIES TIMEOUT 120)
2020-12-08 23:59:52 For more information, please refer to:https://github.com/PaddlePaddle/Paddle/wiki/PaddlePaddle-Unit-test-specification
2020-12-08 23:59:52 There are 1 approved errors.
```
I see this unit-test run very fast
```
2020-12-09 00:57:02 115/215 Test  #232: test_analyzer_multi_model_prediction .............   Passed    7.73 sec
```
You can remove line 650

Reply:
done